lib/features/mailbox/data/network/mailbox_api.dart

```diff
@@ -59,6 +61,23 @@ class MailboxAPI with HandleSetErrorMixin {
     return MailboxResponse(mailboxes: resultCreated?.list, state: resultCreated?.state);
   }
 
+  Set<CapabilityIdentifier>  capabilitiesSupportedMailboxes(Session session, AccountId accountId) {
+    final getMailboxCreated = GetMailboxMethod(accountId);
+    try {
+     requireCapability(
```

Comment:
IMO, Only need check capability `CapabilityIdentifier.jmapTeamMailboxes`
```suggestion
requireCapability(
  session,
  accountId,
  [CapabilityIdentifier.jmapTeamMailboxes]
);
```